indexprovider/wrapper_test.go

```diff
@@ -338,26 +326,15 @@ func TestUnsealedStateManagerStateChange(t *testing.T) {
 			minerID, err := address.IDFromAddress(deals[0].ClientDealProposal.Proposal.Provider)
 			require.NoError(t, err)
 
-			// Set the first response from MinerAPI.StorageList()
-			resp1 := tc.storageListResponse1(abi.SectorID{Miner: abi.ActorID(minerID), Number: deals[0].SectorID})
+			// Set the current state from db -- response from MinerAPI.StorageList()
+			resp1 := tc.initialState(abi.SectorID{Miner: abi.ActorID(minerID), Number: deals[0].SectorID})
 			storageMiner.storageList = map[storiface.ID][]storiface.Decl{}
 			if resp1 != nil {
 				storageMiner.storageList["uuid"] = []storiface.Decl{*resp1}
 			}
 
-			// Trigger check for updates
```

Comment:
Looks like previously each test case was doing 2 updates, and not 1 update. At the moment I have it refactored to do 1 update per test, let's decide if we want another 10 tests for the other cases.